.gitattributes

```diff
@@ -5,3 +5,5 @@
 .travis.yml export-ignore
 phpcs.xml export-ignore
 phpunit.xml.dist export-ignore
+docker-compose.yml export-ignore
+Dockerfile export-ignore
```

Comment:
Missed it before: we need to have here the whole `/.docker/` directory, not just the `Dockerfile`.

Reply:
Done following test example in same file